command/job_periodic_force.go

```diff
@@ -0,0 +1,152 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/nomad/api"
+
+	"github.com/posener/complete"
```

Comment:
the new line here is also odd.  Might want to inspect your gofmt/goimport version - we don't currently distinguish between nomad and 3rd party imports in grouping.